apps/docs/src/components/HomeLayout/Content/index.tsx

```diff
@@ -41,7 +36,7 @@ export default Content;
 const styles = StyleSheet.create({
 	container: {
```

Comment:
I think this "container" style could be in `utils/style.ts` like what we did in Landing page, make it reuse-able